thinc/compat.py

```diff
@@ -32,6 +32,7 @@
 
     has_torch = True
     has_torch_cuda_gpu = torch.cuda.device_count() != 0
+    has_torch_mps = hasattr(torch, "has_mps") and torch.backends.mps.is_built()
```

Comment:
Why do we need a distinction between `has_torch_mps` and `has_torch_mps_gpu`? Also, don't we need to use [`torch.backends.mps.is_available()`](https://pytorch.org/docs/stable/backends.html#module-torch.backends.mps) in conjunction with `is_built()`?

On a related note, since the `has_mps` attribute appears to be undocumented, don't we want to stick to the functions in `torch.backends.mps` instead?

Reply:
Fixed.